storage/acl.js

```diff
@@ -0,0 +1,209 @@
+// limitations under the License.
+
+'use strict';
+
+// [START all]
+// [START setup]
+// By default, the client will authenticate using the service account file
+// specified by the GOOGLE_APPLICATION_CREDENTIALS environment variable and use
+// the project specified by the GCLOUD_PROJECT environment variable. See
+// https://googlecloudplatform.github.io/gcloud-node/#/docs/google-cloud/latest/guides/authentication
+var Storage = require('@google-cloud/storage');
+
+// Instantiate a storage client
+var storage = Storage();
+// [END setup]
+
+// [START add_access_control]
+/**
+ * Add access controls to a bucket or file.
+ *
+ * @param {object} options Configuration options.
+ * @param {string} options.bucket The bucket for the new access controls.
+ * @param {string} options.entity The entity for the new access controls.
+ * @param {string} options.role The role for the new access controls.
+ * @param {string} [options.file] Optional. The file for the new access controls.
+ * @param {boolean} [options.default] Optional. Whether to set default access controls.
+ * @param {function} callback The callback function.
+ */
+function addAccessControl (options, callback) {
+  // Reference the specified storage bucket
+  var bucket = storage.bucket(options.bucket);
+  // Reference the bucket's acl resource
+  var acl = bucket.acl;
+
+  if (options.file) {
+    // Optionally target a file's acl resource
+    acl = bucket.file(options.file).acl;
+  } else if (options.default) {
+    // Optionally add "default" access controls to the bucket
+    acl = acl.default;
+  }
+
+  // Specify the entity and role for the new access control object
+  var config = {
+    entity: options.entity,
+    role: options.role
+  };
+
+  acl.add(config, function (err, aclObject) {
+    if (err) {
+      return callback(err);
+    }
+
+    console.log('Added access controls to: gs://%s' + (options.file ? '/%s' : ''), options.bucket, options.file || '');
+    return callback(null, aclObject);
+  });
+}
+// [END add_access_control]
+
+// [START get_access_control]
+/**
+ * Get access controls for a bucket or file.
+ *
+ * @param {object} options Configuration options.
+ * @param {string} options.bucket The bucket to target.
+ * @param {string} [options.entity] Optional. The entity to filter by.
+ * @param {string} [options.file] Optional. The file to target.
+ * @param {boolean} [options.default] Optional. Whether to get default access controls.
+ * @param {function} callback The callback function.
+ */
+function getAccessControl (options, callback) {
+  // Reference the specified storage bucket
+  var bucket = storage.bucket(options.bucket);
+  // Reference the bucket's acl resource
+  var acl = bucket.acl;
+
+  if (options.file) {
+    // Optionally target a file's acl resource
+    acl = bucket.file(options.file).acl;
+  } else if (options.default) {
+    // Optionally get "default" access controls for the bucket
+    acl = acl.default;
```

Comment:
I added links to in the source code relevant gcloud-node methods (with the "latest" version in the URL path).